public_html/wp-content/mu-plugins/blocks/source/blocks/speakers/block-content.js

```diff
@@ -157,16 +151,13 @@ export class BlockContent extends Component {
 
 		return (
 			<PostList
-				{ ...this.props }
+				attributes={ attributes }
 				className="wordcamp-speakers"
 			>
 				{ posts.map( ( post ) =>
 					<div
 						key={ post.slug }
-						className={ classnames(
-							'wordcamp-speakers__post',
-							`slug-${ post.slug }`,
-						) }
+						className={ `wordcamp-speakers__post slug-${ post.slug }` }
```

Comment:
I like removing `classnames` 👍 It almost always seems silly to be to be dependent on 3rd party code just to concat some strings.